cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -26,6 +26,7 @@
  * The util class for Kafka Cruise Control servlet.
  */
 public class KafkaCruiseControlServletUtils {
+  // FIXME: Read this from a configuration
```

Comment:
I think one way to achieve this would be the following:

(1) Delete `public static final String REQUEST_URI = "/KAFKACRUISECONTROL/";` from this class.
(2) Change: `ParameterUtils#endPoint(HttpServletRequest request)` -> `ParameterUtils#endPoint(HttpServletRequest request, String requestUri)` and handle `requestUri` with ending `*` in this function.
(3) Explicitly pass the requestUri to retrieve the endpoint from the new function in step-2.

Reply:
i will address this later as the config is not available in this class.